contracts/lib/ScaleValidations.sol

```diff
@@ -0,0 +1,39 @@
+/*
+    Copyright 2019 Set Labs Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+pragma solidity 0.5.7;
+
+import { SafeMath } from "openzeppelin-solidity/contracts/math/SafeMath.sol";
+
+
+library ScaleValidations {
+    using SafeMath for uint256;
+
+    uint256 private constant ONE_HUNDRED_PERCENT = 10 ** 18;
```

Comment:
One thing I like the ABDK guys suggested was to start using `1e14`. Didn't realize that was a pattern you could use in Solidity.